vpd-tool/include/vpd_tool.hpp

```diff
@@ -39,5 +39,32 @@ class VpdTool
                     const std::string& i_recordName,
                     const std::string& i_keywordName, const bool i_onHardware,
                     const std::string& i_fileToSave = {});
+
+    /**
+     * @brief Write keyword's value.
+     *
+     * API to update VPD keyword's value to the given input path.
+     * If i_onHardware value in true, i_fruPath is considered has hardware path
```

Comment:
> i_fruPath is considered has hardware path

There is no input parameter `i_fruPath` in the API?
Did you mean "`i_vpdPath` is considered as hardware path"?